### PR TITLE
[MIG] connector_ecommerce: Migration to 10.0

### PR DESCRIPTION
**Pull request by esousy**

### CODE REVIEW

**Comment by esousy** [comment]
I already sent a CLA request and I'm still waiting ...
Can someone help?

**Comment by pedrobaeza** [comment]
Please check Travis status

**Comment by esousy** [comment]
I got this error:

UserError: (u"You try to install module 'connector_ecommerce' that depends on module 'connector'.\nBut the latter module is not available in your system.", None)

the module 'connector' exists in my repo, but doesn't appears in addons path:
Using repo odoo/odoo and addons path /home/travis/build/esousy/connector-ecommerce,/home/travis/dependencies/bank-payment,/home/travis/dependencies/product-attribute,/home/travis/dependencies/account-closing,/home/travis/dependencies/server-tools,/home/travis/dependencies/stock-logistics-warehouse,/home/travis/dependencies/stock-logistics-barcode,/home/travis/odoo-10.0/addons

Any idea?

**Comment by pedrobaeza** [comment]
The module connector is not yet migrated AFAIK.

**Comment by esousy** [comment]
Oops, 
How can I help?
I already migrated it, may I do a new pull request for connector module?

**Comment by pedrobaeza** [comment]
It's being split into several parts. Check https://github.com/OCA/connector/issues/206

**Comment by lasley** [comment]
@esousy - Also https://github.com/OCA/queue/pull/1

There's a working 1:1 in https://github.com/OCA/connector/pull/211, but the API will not be the same as with the finished product - noted in the comments.

File: README.md [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
-*OpenERP Connector is a powerful framework to develop any kind of bi-directional connector between OpenERP (Open Source ERP) and any other software or service.*
+*Odoo Connector is a powerful framework to develop any kind of bi-directional connector between Odoo (Open Source ERP) and any other software or service.*
 
-This OpenERP add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
+This Odoo add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
 
 This project contains modules extending the connector for projects dealing with E-Commerce.
 
-The team for the community is here: https://launchpad.net/~openerp-connector-community
+The team for the community is here: https://launchpad.net/~odoo-connector-community
```
Comment (bealdav) [inline README.md:13]: really yet in lp ?

File: connector_ecommerce/models/invoice.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # © 2013 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
-from openerp import models, api
-from openerp.addons.connector.session import ConnectorSession
+from odoo import models, api
+from odoo.addons.connector.session import ConnectorSession
```
Comment (yk2kus) [inline connector_ecommerce/models/invoice.py:6]: no session in connect in V10

**Comment by lmignon** [comment]
@esousy Do you still work on this PR? If it's not the case I'll finalize your work into a new one based on yours.

**Comment by guewen** [comment]
Work has been finalized and merged in #39, hence I close it. Thanks for your work.

**Pull request closed without merging**